core/src/main/java/org/renjin/primitives/Sort.java

```diff
@@ -239,22 +271,24 @@ public static IntVector psort(IntVector x, Vector indexes) {
 
 
   @Internal
-  public static LogicalVector qsort(LogicalVector x, boolean returnIndexes) {

```

Comment:
Why is `returnIndexes` a `boolean` here while it is defined as `LogicalVector` on the two other methods?